maxim/logger/openai/completions.py

```
@@ -6,10 +6,54 @@
+                        self._generation.result(combined_response)
+                    if self._is_local_trace and self._trace is not None:
+                        # Extract combined text from chunks
+                        combined_text = "".join(
+                            choice.delta.content or ""
+                            for chunk in self._chunks
+                            for choice in chunk.choices
+                            if hasattr(choice.delta, "content")
+                        )
+                        self._trace.set_output(combined_text)
+                        self._trace.end()
+                except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)